### PR TITLE
Update function support list

### PR DESCRIPTION
**Pull request by IchordeDionysos**
I've checked some top functions if they are supported. Stopped at `\leftharpoondown`.

What I've discovered was that there were some functions already supported, but not properly displayed.
Some errors/strange behavior I discovered:

- Displayed italic and not upright. (Examples: `\vert`, `\vee`, `\forall`, `\rangle`, `\langle`, `\cup`, ...)
- Displayed bold and not slim (Examples: `\leq`, `\forall`, ...)
- Cut off (Examples: `\mu`, `\forall`, ...)
- Char modification displayed, but next to not over char (`\bar`, `\wedge`, ...)

---

I've also moved `{align}`/`{aligned`/ ... to `\begin` and `\end` where they are used with.

### CODE REVIEW

**Comment by IchordeDionysos** [comment]
Not sure if all of those functions and those working, but maybe not perfect should be marked as done.

I think, I've marked only functions as done, which do not fall into one of the above categories.

**Comment by creativecreatorormaybenot** [comment]
Regarding the bold and slim - please add to #30 

Regarding the support → there is a problem with ambiguity because some commands can be used as symbol (`\sum`) and as functions (`\sum{..}{..}`)

It would be great if this PR could wait until #29 is fixed (the issue also includes not trying to Pub publish when there was no Pubspec update) - want to do that tomorrow 🙂

**Comment by creativecreatorormaybenot** [comment]
@IchordeDionysos Ready 😃

File: function_prioritization.csv [hunk 1]
```diff
@@ -28,19 +32,19 @@ function,supported,frequency
 \int,true,437
 \operatorname,false,431
 \dfrac,false,353
-\bar,false,315
+\bar,true,315
```
Comment (IchordeDionysos) [inline function_prioritization.csv:35]: Symbol supported, but not as function

File: function_prioritization.csv [hunk 2]
```diff
@@ -112,15 +116,15 @@ function,supported,frequency
...
-\triangle,false,8
+\triangle,true,8
 \tau,true,8
 \small,false,8
 \cdots,false,7
-\partial,false,7
+\partial,true,7
 \rfloor,false,7
-\tilde,false,7
+\tilde,true,7
```
Comment (IchordeDionysos) [inline function_prioritization.csv:125]: Symbol supported, but not as function

File: function_prioritization.csv (hunk 1 above)
Comment (creativecreatorormaybenot) [inline function_prioritization.csv:35]:
```suggestion
\bar,false,315
```

**Pull request merged**